File: skills/pharaoh-api-coverage-check/fixtures/go-fully-covered/input-source.go
// Package inventory demonstrates the go public-symbol regex row.
package inventory

type Catalog struct {
	Name string
}

type Reader interface {
	Read() string
}

func LoadCatalog(path string) *Catalog {
	return &Catalog{Name: path}
}

func SaveCatalog(cat *Catalog, path string) bool {
	_ = cat
	_ = path
	return true
}

func internalHelper() int {
	return 42
}

type privateState struct {
	token string
}

File: skills/pharaoh-api-coverage-check/fixtures/java-fully-covered/input-source.java
package com.example.inventory;

public class Catalog {
    public String name;
}

public interface Reader {
    String read();
}

public class CatalogService {
    public int loadCatalog(String path) {
        if (path == null) {
            throw new CatalogError("path required");
        }
        return 0;
    }

    public int saveCatalog(String path) {
        if (path == null) {
            throw new InvalidPathError("path required");
        }
        return 0;
    }
}

class InternalHelper {
    int compute(int x) { return x; }
}
